my-sms/Gemfile

```
@@ -0,0 +1,43 @@
+source 'https://rubygems.org'
+
+gem 'rails', '3.2.0'
```

To make sure you get the most recent bugfixes related to your rails version you can use ("~>"):
```suggestion
gem 'rails', '~> 3.2.0'
```
The patch version (the x in 3.2.x) is associated with important bugfixes. If you run `bin/bundle update rails`, you should notice the patch version change. Alternatively, you can delete Gemfile.lock and just run `bin/bundle install` instead. See all rails versions: https://rubygems.org/gems/rails/versions